Tools/Scripts/libraries/webkitbugspy/webkitbugspy/mocks/bugzilla.py

```diff
@@ -54,7 +55,7 @@ def __init__(self, hostname='bugs.example.com', users=None, issues=None, environ
 
         prefix = self.hosts[0].replace('.', '_').upper()
         self._environment = environment or mocks.Environment(**{
-            '{}_USERNAME'.format(prefix): 'usernmae',
+            '{}_USERNAME'.format(prefix): 'username',
```

Comment:
Note: I added a drive-by fix here. I assume this is not doing anything, because it doesn't seem to affect test results.